src/integrations/feed/configs.ts

```diff
@@ -0,0 +1,80 @@
+import { FeedConfig, FeedConfigGenerator } from './types';
+import { feedToFilters } from '../../common';
+
+type Options = {
```

Comment:
Should we not also have something for old config: `removeHiddenPosts`
And `allowSquadPosts` (the public ones)

Reply:
It's not supported by feed service at the moment